.github/workflows/add-trailer.yml

```
@@ -0,0 +1,128 @@
+  workflow_run:
+    workflows: ["Review"]
+    types: [completed]
+
+permissions:
+  contents: read
+  pull-requests: read
+  actions: read
+
+jobs:
+  add-trailer:
+    if: |
```

actionlint (1.7.8)

if: condition "${{ (github.event_name == 'issue_comment' && github.event.issue.pull_request\n    && (contains(github.event.comment.body, 'Acked-by:') || contains(github.event.comment.body, 'Tested-by:')))\n|| (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)